recovery/translation_zh_TW.ts

```diff
@@ -1787,10 +1787,10 @@ Partition not big enough for new image.</source>
         <source>repo_list=</source>
         <translation>repo_list=</translation>
     </message>
-    <message utf8="true">
+    <message>
         <location filename="optionsdialog.ui" line="312"/>
         <location filename="optionsdialog.ui" line="339"/>
-        <source> Similar to alt_image_source but it matches the NOOBS method of specifying a list of additional respositories as a space separated quoted string.</source>
+        <source> Similar to&#xa0;alt_image_source&#xa0;but it matches the NOOBS method of specifying a list of additional respositories as a space separated quoted string.</source>
```

Comment:
Where are these `&#xa0;` coming from? @procount ?

Reply:
Dunno. Is it a back-tick?

Reply:
Yeah, back-ticks that got copy-pasted from the markdown documentation, but they don't show up in QT. I'll remove them when I merge. Thanks

Reply:
https://www.google.com/search?q=xa0 says it's a non-breaking space, i.e. you can't "see" it, but Qt is picking up that it's different to a normal space. You probably want to edit them out of `optionsdialog.ui` ?
And it probably doesn't hurt to remove the space at the front of the string either? :wink: